examples/advanced_agents/sandbox_agent/src/nat_sandbox_agent/tools/sandbox/execution.py

```
@@ -0,0 +1,174 @@
+        timeout: Optional timeout override.
+
+    Returns:
+        Dict with status, stdout, stderr, and generated files.
+    """
+    logger.info(f"Executing Python code ({len(code)} chars)")
+
+    # Write code to a temp file for better error messages
+    try:
+        await executor.sandbox.write_file(DEFAULT_SCRIPT_PATH, code)
+    except Exception as e:
+        logger.exception(f"Failed to write script file: {e}")
```

Ruff (0.14.13)

Redundant exception object included in `logging.exception` call

(TRY401)